### PR TITLE
Http errors handling

### PR DESCRIPTION
**Pull request by r-erema**

### CODE REVIEW

File: internal/http/image.go [hunk 1]
```diff
@@ -37,27 +38,35 @@ func NewImageHandler(
 	}
 }
 
-// ServeHTTP sends message with client`s image to WhatsApp server.
-func (h *SendImageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (h *SendImageHandler) Handle(w http.ResponseWriter, r *http.Request) *AppError {
```
Comment (vetheslav) [inline internal/http/image.go:41]: Add comment

File: internal/http/image.go [hunk 2]
```diff
@@ -37,27 +38,35 @@ func NewImageHandler(
...
 	decoder := json.NewDecoder(r.Body)
 	var msgReq SendImageRequest
 	err := decoder.Decode(&msgReq)
 	if err != nil {
-		handleError(w, "can't decode request", err, http.StatusBadRequest)
-		return
+		return &AppError{
```
Comment (vetheslav) [inline internal/http/image.go:46]: 1) Add fields when you use structures. It's difficult to read your code with structures in Github
2) Maybe it's better to create a function-helper to create an error. But it up to you

Comment (r-erema) [reply internal/http/image.go:46]: I think this simple operation doesn’t deserve separate function yet:)

File: internal/http/image.go (hunk 1 above)
Comment (r-erema) [reply internal/http/image.go:42]: Done

**Pull request merged**